openmc/element.py

```diff
@@ -45,9 +45,9 @@ def __init__(self, name='', xs=None):
 
     def __eq__(self, other):
         if isinstance(other, Element):
-            if self._name != other._name:
+            if self._name != other.name:
```

Comment:
Sure. I had only changed the other one because it was a more egregious sin to access the private attribute of anything other than `self`, but we can change `self` as well.